lms/templates/navigation.html

```diff
@@ -123,35 +123,40 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
           </li>
         % endif
       </%block>
-      % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
-          % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
-          <li class="nav-global-05">
-            <a class="cta cta-discovery" href="/courses">${_("Find Courses")}</a>
-          </li>
-          %endif
-          % if course and settings.FEATURES.get('RESTRICT_ENROLL_BY_REG_METHOD') and course.enrollment_domain:
-          <li class="nav-global-04">
-            <a class="cta cta-register" href="${reverse('course-specific-register', args=[course.id.to_deprecated_string()])}">${_("Register")}</a>
-          </li>
-          % else:
-          <li class="nav-global-04">
-            <a class="cta cta-register" href="/register${login_query()}">${_("Register")}</a>
-          </li>
-          % endif
 
-      % endif
+      <%block name="navigation_other_global_links">
+        % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
```

Comment:
Can we take this opportunity to move these settings references out of the template and into the view?  The logic can be used to set boolean context parameters for displaying the elements.